MJPEG streaming (#1352)
cyberbotics/webots: docs/guide/web-streaming.md

In this mode the user can watch an already running simulation and navigate into the scene but cannot interact with the simulation, i.e. controlling the execution of the simulation and modifying the robot controller program.
Please refer to the [Web simulation](web-simulation.md) documentation to setup a platform where users can run individual simulations.

Two different streaming modes are available:

> - `x3d` mode (default) where the simulation is streamed using a mechanism similar to the [web animation export](web-animation.md) except that the `X3D` file and the animation are sent on the fly to the web browser clients.

Reviewer: ```suggestion
- `x3d` (default): the simulation is streamed using a mechanism similar to the [web animation export](web-animation.md) except that the `X3D` file and the animation are sent on the fly to the web browser clients.
```

> - `mjpeg` mode where the Webots scene rendered on the server is directly sent as `MJPEG` image to the web browser client.

Reviewer: ```suggestion
- `mjpeg`: the Webots scene rendered on the server is directly sent as an `MJPEG` image to the web browser client.
```
